app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,5 @@
+import dayjs from "lib/dayjs";
```

Comment:
This is wrong. I want you to carefully go through https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F

Once done, please create a loom video and point out what's the mistake here and how to fix it.

Reply:
It's done,
[Loom Video](https://www.loom.com/share/ae3338347e9b4249bc3a7b1089af732b).
I will be pushing the changes soon.

Reply:
@shivamkrs89 Yes. Correct.

But in order to further solidify your understanding, you can read the following:

- Singletons: https://rajeshnaroth.medium.com/singletons-in-javascript-and-es6-26f276a21df8#:~:text=The%20ES6%20Way%20%E2%80%94%20modules
- https://github.com/iamkun/dayjs/blob/dev/src/index.js#L395-L401